source/speech/__init__.py

```diff
@@ -1606,15 +1606,12 @@ def getPropertiesSpeech(  # noqa: C901
 	if rowCount or columnCount:
 		# The caller is entering a table, so ensure that it is treated as a new table, even if the previous table was the same.
 		oldTableID = None
-	ariaCurrent = propertyValues.get('current', False)
-	if ariaCurrent:
-		try:
-			ariaCurrentLabel = controlTypes.isCurrentLabels[ariaCurrent]
-			textList.append(ariaCurrentLabel)
-		except KeyError:
-			log.debugWarning("Aria-current value not handled: %s"%ariaCurrent)
-			ariaCurrentLabel = controlTypes.isCurrentLabels[True]
-			textList.append(ariaCurrentLabel)
+
+	# speak isCurrent property EG aria-current
+	isCurrent = propertyValues.get('current', controlTypes.IS_CURRENT.NO)
+	if isCurrent != controlTypes.IS_CURRENT.NO:
+		textList.append(controlTypes.IS_CURRENT.getTranslationForUI(isCurrent))

```

Comment:
I think this could be 
```suggestion
		textList.append(isCurrent.getTranslationForUI())
```